tests/client/amo/components/TestAddonReview.js

```diff
@@ -92,8 +94,8 @@ describe('AddonReview', () => {
           apiState: signedInApiState,
         });
 
-        // Make sure the overlay was hidden after finishing.
-        assert.ok(overlayCard.hide.called);
+        assert.ok(onReviewSubmitted.called,
+                  'onReviewSubmitted() should have been called after updating');
```

Comment:
Super nitpick but I think we're sticking to just two-space indentation rather than alignment.